modules/bibdk_reservation/includes/bibdk_reservation_form_step_2.inc

```diff
@@ -61,15 +61,17 @@ function bibdk_reservation_form_step_2($form, &$form_state) {
       $form['messages_wrapper']['messages'] = $msgs;
     }
 
-    if ($reason = $orderpolicy->getOrderPossibleReason()){
-      if ($reason != 'not_owned_accepted_by_consortia' and $show_url == true)
+    if ($reason = $orderpolicy->getOrderPossibleReason()) {
+      if ($reason != 'not_owned_accepted_by_consortia' and $show_url == true) {
```

Comment:
I know 'and' is valid code - but we use && all the other places. Consistency ;-)